ocsmesh/cli/remesh.py

```diff
@@ -306,61 +309,55 @@ def run(self, args):
 
         _logger.info("Clip mesh by inverse of region of interest")
         fixed_mesh_w_hole = utils.clip_mesh_by_shape(
-            jig_init, region_of_interest, fit_inside=True, inverse=True)
+            jig_init, region_of_interest, fit_inside=True, inverse=True
+        )
 
-        _logger.info(
-                "Get all initial mesh vertices in the region of interest")
-        vert_idx_to_refin = utils.get_verts_in_shape(
-            jig_hfun, region_of_interest)
+        _logger.info("Get all initial mesh vertices in the region of interest")
+        vert_idx_to_refin = utils.get_verts_in_shape(jig_hfun, region_of_interest)
 
-        fixed_mesh_w_hole.point['IDtag'][:] = -1
-        fixed_mesh_w_hole.edge2['IDtag'][:] = -1
+        fixed_mesh_w_hole.point["IDtag"][:] = -1
+        fixed_mesh_w_hole.edge2["IDtag"][:] = -1
 
         refine_opts = jigsawpy.jigsaw_jig_t()
         refine_opts.hfun_scal = "absolute"
         refine_opts.hfun_hmin = np.min(jig_hfun.value)
         refine_opts.hfun_hmax = np.max(jig_hfun.value)
         refine_opts.mesh_dims = +2
         # Mesh becomes TOO refined on exact boundaries from DEM
-#    refine_opts.mesh_top1 = True
-#    refine_opts.geom_feat = True
+        #    refine_opts.mesh_top1 = True
+        #    refine_opts.geom_feat = True
```

Comment:
Please remove extra spaces between # and the commented code following it now that the lines are shifted